lib/tasks/test.js

```diff
@@ -264,5 +271,45 @@ module.exports.browserTest = function(gulp, config) {
 	});
 };
 
+module.exports.pa11yTest = function(gulp, config) {
+	const test = pa11y({
+		log: {
+			debug: console.log.bind(console),
+			error: console.error.bind(console),
+			info: console.info.bind(console)
+		},
+		ignore: ['WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchID']
+	});
+
+
+	return new Promise(function(resolve, reject) {
+		config = config || {};
+		let src = (config.pa11yPath !== undefined) ? path.join(process.cwd(), config.pa11yPath) : path.join(process.cwd(), '/demos/local/pa11y.html');
```

Comment:
Probably worth doing this right at the top of the subtask, so you can then check if the file exists, and if not, stop doing anything. Might also be worth not requiring pa11y until this check has been made as requires is what slows down obt the most